api/server/sdk/volume_ops.go

```diff
@@ -115,11 +116,17 @@ func (s *VolumeServer) Create(
 			"Must supply spec object")
 	}
 
-	spec := req.GetSpec()
 	locator := &api.VolumeLocator{
 		Name: req.GetName(),
 	}
 	source := &api.Source{}
+	// Update given spec according to enforcement policy set
+	// In case policy is not set, should fall back to default way
+	// of creating volume
+	spec, err := GetEnforcedVolSpecs(locator, source, req.GetSpec())
```

Comment:
I thought this was going to be in the driver. If it is here, that is find. Just add to the Docker API and CSI API also.